tilavarauspalvelu/settings.py

```diff
@@ -602,6 +607,46 @@ def get_git_revision_hash() -> str:
 OPEN_CITY_PROFILE_LEVELS_OF_ASSURANCES = env("OPEN_CITY_PROFILE_LEVELS_OF_ASSURANCES")
 PREFILL_RESERVATION_WITH_PROFILE_DATA = env("PREFILL_RESERVATION_WITH_PROFILE_DATA")
 
+# Configure Redis cache
+#
+# Contains conditional configuration because OpenShift and local
+# environments are different
+if env("REDIS_URL"):
+    SESSION_ENGINE = "django.contrib.sessions.backends.cache"
+    SESSION_CACHE_ALIAS = "default"
+
+    # Redis sentinel is used in OpenShift
+    if env("REDIS_SENTINEL_SERVICE") and env("REDIS_MASTER"):
```

Comment:
Looks like these settings does not depend on the `REDIS_URL` . We could move this whole if block at same level after the [line](https://github.com/City-of-Helsinki/tilavarauspalvelu-core/pull/830/files#diff-0e8763694e016724bf39fd6f01d866632fc493c07ed173914c32345722d67d6cR614) `if env("REDIS_URL"):` as overriding functionality and to reduce nesting.